cloudslang-cli/src/main/java/io/cloudslang/lang/cli/utils/MetadataHelperImpl.java

```diff
@@ -63,8 +69,28 @@ private Metadata cleanWhiteSpaces(Metadata metadata, String lineBreak) {
     }
 
     private void cleanMapWhiteSpaces(Map<String, String> map, String lineBreak) {
-        for (String key : map.keySet()) {
-            map.put(key, map.get(key).trim().replaceAll("[\\s]+" + lineBreak, lineBreak));
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            map.put(entry.getKey(), entry.getValue().trim().replaceAll("[\\s]+" + lineBreak, lineBreak));
```

Comment:
is it possible for the value to be null?

better use StringUtils or check yourself